Podfile

```diff
@@ -222,7 +222,7 @@ abstract_target 'Apps' do
 
   pod 'Gridicons', '~> 1.1.0'
 
-  pod 'WordPressAuthenticator', '~> 2.0.0'
+  pod 'WordPressAuthenticator', '~> 2.1.0'
```

Comment:
Should we make this `~> 2.1` instead of `~> 2.1.0`, so that SemVer would work (as the `~>` operator allows the last specified digit to increase but not the others, so `~> 2.1.0` would allow `2.1.1` but not `2.2.0`, while `~> 2.1` would.

Reply:
Note that we should probably also consider doing the same (using `~> x.y` instead of `~> x.y.z`) for all other dependencies in the `Podfile` as well, though that larger/broader change for all pods would probably be for a separate PR.